### PR TITLE
Update README file ro remove JWST refs and regression test examples, Rcal 65

### PR DESCRIPTION
**Pull request by ddavis-stsci**
Updated the changes file and removed JWST refs from README

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+0.0.1 (unreleased)
```
Comment (dmggh) [inline CHANGES.rst:1]: Should that be 0.0.0 ?

**Comment by ddavis-stsci** [comment]
I'm assuming that our first build will be 0.0.1 but I'm not sure. I was 
using 0.0.0 as a placeholder.
I should be the same in any case.

**Review by sosey — CHANGES_REQUESTED** [review]

**Comment by ddavis-stsci** [comment]
Closing this PR, the changes here were include in PR #70

**Comment by ddavis-stsci** [comment]
Closing this PR, the changes here were include in PR #70

**Pull request closed without merging**